src/server/Server.java

```diff
@@ -159,46 +145,24 @@ private SSLContext getSSLContext(Path keyStorePath, char[] keyStorePass, String
         return sslContext;
     }
 
-    public HashMap<String, String> pathFiltration(HashMap<String, String> Headers) {
-        String path = "./" + WWWDir + PathFilter.filter(Headers.get("path"));
```

Comment:
PathFilter is used to mitigate LFI attacks

Reply:
Please reverse the deletion and use the function in ServeFiles class

Reply:
> PathFilter is used to mitigate LFI attacks

Shouldn't the user implement it themselves

Reply:
Nope, it is a critical issue

Reply:
I really don't want to understand it and adapt it to the new interface, can you do it.